src/selectivity-base.js

```diff
@@ -181,6 +181,8 @@ function Selectivity(options) {
 
     this.setOptions(options);
 
+    this.$el.attr('tabindex', 1);
```

Comment:
I think the tab index should at least be configurable. Also, I'm afraid using tab indices might break tab ordering for people who aren't explicitly setting `tabindex`.